week5/src/com/JH/Task2/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.JH.Task2;
+/*
+ *@BaselObaid
+ */
+import java.io.FileInputStream;
+
+public class Main {
+
+    public static void main(String [] args){
+
+        String fileNameDirectory = "C:\\Users\\basel_c8nyn\\OneDrive - Al Azhar University\\Desktop\\week5\\src\\com\\JH\\Task2\\Lincoln.txt";
+
+        try{
+            FileInputStream file = new FileInputStream(fileNameDirectory);
+            int counter  = 0;
+            int c = 0 ;
+            int i=0;
+
+
+            while ((i= file.read())!=-1){
+                if((char)i == ' '|| (char)i == '\n'){
```

Comment:
it counts the lines and spaces what if there is an empty lines or white spaces ?